RecoParticleFlow/PFProducer/plugins/ChargedHadronPFTrackIsolationProducer.cc

```diff
@@ -0,0 +1,101 @@
+/*
+ * ChargedHadronPFTrackIsolationProducer
+ *
+ * Author: Andreas Hinzmann
+ *
+ * Associates PF isolation flag to charged hadron candidates
+ *
+ */
+
+#include "FWCore/Framework/interface/stream/EDProducer.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/ParameterSet/interface/ConfigurationDescriptions.h"
+#include "FWCore/ParameterSet/interface/ParameterSetDescription.h"
+
+#include "FWCore/Utilities/interface/Exception.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+
+#include "DataFormats/ParticleFlowCandidate/interface/PFCandidate.h"
+#include "DataFormats/ParticleFlowCandidate/interface/PFCandidateFwd.h"
+#include "DataFormats/Common/interface/ValueMap.h"
+#include "DataFormats/ParticleFlowReco/interface/PFBlock.h"
+#include "DataFormats/ParticleFlowReco/interface/PFBlockElement.h"
+
+namespace edm {
+   class ConfigurationDescriptions;
+}
+
+class ChargedHadronPFTrackIsolationProducer : public edm::stream::EDProducer<> 
+{
+public:
+
+  explicit ChargedHadronPFTrackIsolationProducer(const edm::ParameterSet& cfg);
+  ~ChargedHadronPFTrackIsolationProducer() {}
+  void produce(edm::Event& evt, const edm::EventSetup& es);
+  static void fillDescriptions(edm::ConfigurationDescriptions & descriptions);
+
+private:
+  // input collection
+  edm::InputTag srcCandidates_;
+  edm::EDGetTokenT<edm::View<reco::PFCandidate> > Candidates_token;
```

Comment:
please keep the same style of local members: start with lower case, end with an underscore